.mergify.yml

```diff
@@ -0,0 +1,18 @@
+pull_request_rules:
+  - name: Automatic strict merge
+    conditions:
+      - base=master
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - "#review-requested=0"
+      - status-success=continuous-integration/travis-ci/pr
```

Comment:
I think this should be "Travis CI - Pull Request" since "continuous-integration/travis-ci/pr" was for the older integration service I think. Not sure.

Reply:
I saw both in their examples and documentation. The service says the config is valid as-is.

Reply:
As a reference point, I got it working when I tested with "Travis CI - Pull Request", I *think* I recall "continuous-integration/travis-ci/pr" not working as expected

Reply:
Updated.